lib/customisation.nix

```
@@ -189,62 +190,76 @@ rec {
     );
 
   /**
-    Call the package function in the file `fn` with the required
-    arguments automatically.  The function is called with the
-    arguments `args`, but any missing arguments are obtained from
-    `autoArgs`.  This function is intended to be partially
-    parameterised, e.g.,
+    Call the function (or the function within the file) `funcOrPath` with `defaultArgs`, which can be overridden by `args`. This function is intended to be partially parameterised / curried (see examples).
+
+    # Inputs (positional)
+
+    - `defaultArgs` - `AttrSet`
+
+      Default
+
+    - `funcOrPath` - `((AttrSet -> a) | Path)`
+
+      Either a function that is expected to build a package from the given attribute set or a file that returns such a function when imported.
+
+    - `args` - `AttrSet`
+
+      Arguments that will be passed to the function. These are merged with `defaultArgs`.
+      Attributes in `args` override or extend those of `defaultArgs`.
```

```suggestion
      Attributes in `args` replace those in `defaultArgs`.
```

Less ambiguous. `extend` could perhaps be misunderstood to merge values.

---

Fixed.